index.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+//require('../')()
+
+const findLinks = require('./utils/findLinks')
+const linksFunctions = require('./utils/validateAndStatusLinks')
+const fs = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+
+const readDirectory=(uri, comandLine4, comandLine5)=>{
```

Comment:
Sugiero que definir mejor los argumentos que recibe la función.
```suggestion
const readDirectory=(path, commandProcess4, commandProcess5)=>{
```